App doesn't boot on web

https://forum.rcpch.tech/t/digital-growth-charts-app-start-here/608/2

```
digital-growth-charts-app/assets/.env:1  Failed to load resource: the server responded with a status of 404 ()
main.dart.js:31896 Error while trying to load an asset: Flutter Web engine failed to fetch "assets/.env". HTTP request succeeded, but the server responded with HTTP status 404.
$1 @ main.dart.js:31896
main.dart.js:28987 AppConfig: CRITICAL ⚠️ .env file load failed: Instance of 'minified:AR'
main.dart.js:28987 Instance of 'minified:Ps'
main.dart.js:28987 at Object.i (https://rcpch.github.io/digital-growth-charts-app/main.dart.js:3642:20)
main.dart.js:28987     at a6I.grr (https://rcpch.github.io/digital-growth-charts-app/main.dart.js:94487:26)
main.dart.js:28987     at MC.L (https://rcpch.github.io/digital-growth-charts-app/main.dart.js:43821:10)
main.dart.js:28987     at wz.iM (https://rcpch.github.io/digital-growth-charts-app/main.dart.js:82606:32)
main.dart.js:28987     at wz.jG (https://rcpch.github.io/digital-growth-charts-app/main.dart.js:82584:9)
main.dart.js:28987     at wz.Dn (https://rcpch.github.io/digital-growth-charts-app/main.dart.js:82532:10)
main.dart.js:28987     at wz.a0s (https://rcpch.github.io/digital-growth-charts-app/main.dart.js:82533:19)
main.dart.js:28987     at wz.Gh (https://rcpch.github.io/digital-growth-charts-app/main.dart.js:82582:11)
main.dart.js:28987     at wz.eF (https://rcpch.github.io/digital-growth-charts-app/main.dart.js:82581:6)
main.dart.js:28987     at xI.rM (https://rcpch.github.io/digital-growth-charts-app/main.dart.js:82445:3)
```

--- a/.github/workflows/deploy-testing.yml
+++ b/.github/workflows/deploy-testing.yml
@@ -99,7 +99,7 @@ jobs:
         uses: actions/configure-pages@v5
 
       - name: Upload artifact for GitHub Pages
-        uses: actions/upload-pages-artifact@v4
+        uses: actions/upload-pages-artifact@v3
         with:
           # Path to the directory containing the static assets to be deployed.
           # Flutter web builds to 'build/web' by default.

